docs/proposals/metrics/README.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Improve Karmada observability
+authors:
+- "@Poor12"
+reviewers:
+- "@TBD"
+approvers:
+- "@TBD"
+
+creation-date: 2022-06-02
+
+---
+
+# Improve Karmada observability
+
+## Summary
+
+Provides metrics to improve Karmada observability
+
+## Motivation
+
+### Goals
+
+- Provides metrics to reflect the status of Karmada resources
+- Provides metrics to reflect the key performance of Karmada components
+- Non-invasive
+- Compliant with Kubernetes metrics specification
+
+### Non-Goals
+
+## Proposal
+
+### User Stories (Optional)
+
+#### Story 1
+
+For now, Karmada lacks the observability of individual components. On the one hand, we need metrics that can reflect the object status in the process of distributing resources to member clusters, such as work and resourceBinding. On the other hand, we need metrics that reflect the performance of Karmada components for upcoming performance test, such as schedule latency.
+
+Now, only Karmada-scheduler has four metrics, `schedule_attempts_total` means numbers of attempts to schedule resourceBindings, `e2e_scheduling_duration_seconds` means E2e scheduling latency in seconds, `scheduling_algorithm_duration_seconds` means scheduling algorithm latency in seconds, `queue_incoming_bindings_total` means numbers of bindings added to scheduling queues by event type.
+
+## Design Details
+
+For object status:
+
+ResourceBinding means delivery situation of the resource template. For resourceBinding, We plan to provide metrics like `karmada_resourcebinding_condition` and `karmada_clusterresourcebinding_condition`. And it will have labels including `name`, `namespace`, `type`.
```

Comment:
```suggestion
ResourceBinding means delivery situation of the resource template. For resourceBinding, we plan to provide metrics like `karmada_resourcebinding_condition` and `karmada_clusterresourcebinding_condition`. And it will have labels including `name`, `namespace`, and `type`.
```